src/main/java/uk/gov/hmcts/reform/orgrolemapping/scheduler/Scheduler.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.orgrolemapping.scheduler;
+
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.orgrolemapping.domain.service.OrganisationService;
+
+@Service
+public class Scheduler {
+
+    private final OrganisationService organisationService;
+
+    public Scheduler(OrganisationService organisationService) {
+        this.organisationService = organisationService;
+    }
+
+    @Scheduled(cron = "${professional.role.mapping.scheduling.cron}")
```

Comment:
see below

Reply:
updated in [9375d38](https://github.com/hmcts/am-org-role-mapping-service/pull/1743/commits/9375d389ba6ce2749085574411c4ea75aa7fd2f6)